# error handling

# try: 
#     code 
# except:
#     code 




# try:
#     x = int(input("enter the value of x = "))
#     y = int(input("enter the value of y = "))
#     ans = x + y 
#     print(ans)
# except:
#     print("enter int only")

# print("thank you")


# ========================================

# try:
#     a = int(input("enter a number = "))
#     b = int(input("enter a number = "))
#     print(a/b)
# except Exception as e:
#     print(e)

